Block/AccountBlockService.php

```diff
@@ -25,7 +25,7 @@
  *
  * @author Thomas Rabaix <thomas.rabaix@sonata-project.org>
  */
-class AccountBlockService extends BaseBlockService
+class AccountBlockService extends AbstractBlockService
```

Comment:
No, because we only use normal block options. There ain't any `buildCreateForm` or `buildEditForm` method.